src/services/mapService.js

```diff
@@ -0,0 +1,63 @@
+export const mapService = {
+  mapSettings: {
+    defaultWidth: 360,
+    defaultHeight: 180,
+    scale: 1.5
+  },
+
+  getMapElement(){
+    return document.getElementById("map");
+  },
+
+  getMapWidth() {
+    return this.mapSettings.defaultWidth * this.mapSettings.scale;
+  },
+
+  getMapHeight(){
+    return this.mapSettings.defaultHeight * this.mapSettings.scale;
+  },
+
+  translateLongitudeToCanvasPx(longitude){
+    let x = (longitude + this.mapSettings.defaultWidth / 2) * this.mapSettings.scale;
+    return x;
+  },
```

Comment:
Why not return directly the result of the formula? Storing the result and returning the result later seems pointless.
Also, the name of the function could be shorter and still express its intent. Try `mapLongitudeToPx`.